R/get_regional_data.R

```diff
@@ -134,20 +138,23 @@ get_regional_data <- function(country, totals = FALSE, include_level_2_regions =
       dplyr::select(date, region_level_2, level_2_region_code, region_level_1, level_1_region_code, 
                     cases_new, cases_total, deaths_new, deaths_total,
                     recovered_new, recovered_total, hosp_new, hosp_total,
-                    tested_new, tested_total) %>%
+                    tested_new, tested_total, dplyr::everything()) %>%
```

Comment:
Does the above variable cleaning work with all the additional variables?

Reply:
Yes, it does